nipype/interfaces/ants/resampling.py

```diff
@@ -369,15 +369,14 @@ class ApplyTransformsInputSpec(ANTSCommandInputSpec):
             traits.Float(), traits.Float()  # Gaussian/MultiLabel (sigma, alpha)
         ),
     )
-    transforms = traits.Either(
-        InputMultiPath(File(exists=True)),
-        "identity",
+    transforms = InputMultiObject(
+        traits.Either(File(exists=True), "identity"),
```

Comment:
In the latest releases of traits they are recommending Union instead of Either. I don't know when Union started to be available or when they started to recommend it though. I'm just writing this down and don't think it requires any particular action in this PR (cc @satra)